docs/sources/flow/reference/components/prometheus.relabel.md

```diff
@@ -29,23 +29,17 @@ each metric in order of their appearance in the configuration file.
 Multiple `prometheus.relabel` components can be specified by giving them
 different labels.
 
-## Example
+## Usage
+
 ```river
-prometheus.relabel "keep_backend_only" {
-	forward_to = [prometheus.remote_write.onprem.receiver]
-
-	rule {
-		action        = "replace"
-		source_labels = ["__address__", "instance"]
-		separator     = "/"
-		target_label  = "host"
-	}
-
-	rule {
-		action        = "keep"
-		source_labels = ["app"]
-		regex         = "backend"
-	}
+prometheus.relabel "LABEL" {
+  forward_to = RECEIVER_LIST
+
+  rule {
```

Comment:
Let's sync this with the discovery.relabel and either have none or both of them displaying dummy rules here?

Reply:
Synced them 👍